src/orders/orders.module.ts

```diff
@@ -0,0 +1,23 @@
+import { Module } from '@nestjs/common';
+import { OrdersService } from './orders.service';
+import { OrdersResolver } from './orders.resolver';
+import { Web3Service } from '../web3/web3.service';
+import { LootBoxesService } from '../loot-boxes/loot-boxes.service';
+import { LootsService } from '../loot-boxes/loots/loots.service';
+import { LootBoxesModule } from '../loot-boxes/loot-boxes.module';
+import { UsersService } from '../users/users.service';
+import { MailService } from '@sendgrid/mail';
+
+@Module({
+  providers: [
+    OrdersService,
+    OrdersResolver,
+    LootBoxesService,
+    LootsService,
```

Comment:
It's cleaner to import the module and have each modules exports the services required. Ex:

```
providers: [
  OrdersModule,
  LootBoxesModules,
]

// inside the LootBox module export the following:
LootBoxesService,
LootsService,
```